eventbrite_saver.py

```diff
@@ -0,0 +1,85 @@
+import build_contentful_data
+import eventsaver
+import os
+import requests
+
+from dotenv import load_dotenv
+from os.path import join, dirname
+from urlparse import urlparse
+
+try:
+    dotenv_path = join(dirname(__file__), '.env')
+    load_dotenv(dotenv_path)
+except Exception as e:
+    print "\nMissing .env file\n"
+
+EVENTBRITE_OAUTH_TOKEN = os.environ.get('EVENTBRITE_OAUTH_TOKEN', None)
+
+EVENTBRITE_API_BASE = 'https://www.eventbriteapi.com/v3/events/'
+
+class EventbriteSaver(eventsaver.EventSaver):
+
+    def __init__(self, url):
+        super(EventbriteSaver, self).__init__(url)
+        self.save_eventbrite_data()
+
+    def get_event_id(self):
+        url_components = urlparse(self.url)
+        if not url_components.netloc == 'www.eventbrite.com':
```

Comment:
I would do the check in the __init__ function and raise an exception if the suffix is not what you expect.